web3.js/src/connection.js

```diff
@@ -1857,6 +1887,35 @@ export class Connection {
       });
   }
 
+  /**
+   * Returns epoch activation information for a stake account that has been delegated
+   */
+  async getStakeActivation(
+    publicKey: PublicKey,
+    commitment: ?Commitment,
+    epoch: ?number,
+  ): Promise<StakeActivationData | null> {
+    let _args = [publicKey.toBase58()];
+    if (epoch !== undefined) {
+      _args.push({epoch});
+    }
+
+    const args = this._buildArgs(_args, commitment, 'base64');
```

Comment:
There should only be on config object here actually and also encoding (`'base64'`) isn't relevant. So if `epoch` and `commitment` are both set, it would look something like this:
```
["<PUBKEY>", { commitment: 'max', epoch: 234234 }]
```

Reply:
thanks, done.